<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="haberList">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1- 1. Haberi Aç</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2- 2. Haber Aç</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3- 3. Haberi Aç</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4- Pencereyi Kapat</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="baslikLbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GÜNCEL HABERLER</string>
    </property>
   </widget>
   <widget class="QLabel" name="haber1Lbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Haber1</string>
    </property>
   </widget>
   <widget class="QLabel" name="haber2Lbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Haber2</string>
    </property>
   </widget>
   <widget class="QLabel" name="haber3Lbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Haber3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1-)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2-)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3-)</string>
    </property>
   </widget>
   <widget class="QLabel" name="newsCountLbl">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="haberArtirBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ARTIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="haberAzaltBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AZALT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="haberOnaylaBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ONAYLA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
